README.md

```diff
@@ -31,13 +31,18 @@ own and can obtain certificates for.
 
 ##### Manual installation
 
-  1. Install Go version 1.10 or higher. Optionally, set
+  1. Install Go version 1.10. Optionally, set
```

Comment:
Google will validate all contributions against Travis running Go 1.10, so should we suggest installing specifically this version?

I've installed Go 1.14, used strings.ReplaceAll and got a build error from Travis:
https://travis-ci.org/github/ampproject/amppackager/builds/680776394

Reply:
@twifkak perhaps it's time to update travis.yml? What do you think?

Reply:
I wouldn't remove the "or higher" unless there's a known breakage at higher versions. Don't want to make installation unnecessarily hard. I guess I have to exclude `-mod=vendor` from that rule. :)

Re: updating travis.yml, SGTM. It may even be possible to test on multiple versions? e.g. https://github.com/streadway/amqp/blob/master/.travis.yml